hermes-console/static/partials/home.html

```diff
@@ -38,4 +37,12 @@
             </a>
         </div>
     </div>
+
+    <div class="row" style="margin-top: 2em;">
+        <div class="col-md-6 col-md-offset-3">
```

Comment:
IMO `col-md-3` would be enough

Reply:
Yes, it's enough, but it looks ugly.

Reply:
I meant changing the importance level of the button to lower (like "stats", "docs"...).
On the other hand I think this just doesn't belong here... Like, constraints are part of console, so we should click "console", then there should be a (new screen maybe), that has two buttons:
- groups / topics / subscriptions
- consumer constraints

Or we can redesign the main screen of console. I have to think more about it. 
Let's keep it as but the button should be displayed for admin users only.

Reply:
I removed this button.